data-raw/process_data_set_alina.R

```diff
@@ -0,0 +1,90 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+library(readr)
+
+# dem_candidates ---------------------------------------------------------------
+dem_candidates <- read_csv("data-raw/primary-candidates-2018/dem_candidates.csv") %>%
+  clean_names() %>%
+  #change classes from character to factor and date to date
+  mutate(
+    state = as.factor(state),
+    district = as.factor(district),
+    office_type = as.factor(office_type),
+    race_type = as.factor(race_type),
+    primary_status = as.factor(primary_status),
+    primary_runoff_status = as.factor(primary_runoff_status),
+    general_status = as.factor(general_status),
+    race = as.factor(race),
+    race_primary_election_date = as.Date(race_primary_election_date,"%m/%d/%y"),
+    won_primary = as.factor(won_primary),
+    veteran = as.factor(veteran),
+    lgbtq = as.factor(lgbtq),
+    elected_official = as.factor(elected_official),
+    self_funder = as.factor(self_funder),
+    stem = as.factor(stem),
+    obama_alum = as.factor(obama_alum),
+    party_support = as.factor(party_support),
+    emily_endorsed = as.factor(emily_endorsed),
+    guns_sense_candidate = as.factor(guns_sense_candidate),
+    biden_endorsed = as.factor(biden_endorsed),
+    warren_endorsed = as.factor(warren_endorsed),
+    sanders_endorsed = as.factor(sanders_endorsed),
+    our_revolution_endorsed = as.factor(our_revolution_endorsed),
+    justice_dems_endorsed = as.factor(justice_dems_endorsed),
+    pccc_endorsed = as.factor(pccc_endorsed),
+    indivisible_endorsed = as.factor(indivisible_endorsed),
+    wfp_endorsed = as.factor(wfp_endorsed),
+    no_labels_support = as.factor(no_labels_support)
+  )
+
+#change levels from Yes/No to TRUE/FALSE for relevant variables
+levels(dem_candidates$won_primary) <- c(FALSE,TRUE)
```

Comment:
Interesting. I'd say that having a `factor` with levels `TRUE` and `FALSE` is a bad idea. I would say either stick with `factor` with levels `Yes`/`No`, or convert to `logical`. What do you think @rudeboybert ?

Reply:
I first converted to a factor with TRUE/FALSE and then converted to logical.

Reply:
OK, great.